{"name": "The user should be able to edit ruleset successfully. -- @1.1 ", "status": "broken", "statusDetails": {"message": "smtplib.SMTPAuthenticationError: (534, b'5.7.9 Please log in with your web browser and then try again. For more\\n5.7.9 information, go to\\n5.7.9  https://support.google.com/mail/?p=WebLoginRequired d9443c01a7336-1f44c9e43c2sm4729765ad.308 - gsmtp')\n", "trace": "  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"Features\\environment.py\", line 108, in after_step\n    s.login(fromaddr, \"E$$pl@2022!#21\")\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 750, in login\n    raise last_exception\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 739, in login\n    (code, resp) = self.auth(\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 662, in auth\n    raise SMTPAuthenticationError(code, resp)\n"}, "steps": [{"name": "And Click on ruleset and edit", "status": "broken", "statusDetails": {"message": "smtplib.SMTPAuthenticationError: (534, b'5.7.9 Please log in with your web browser and then try again. For more\\n5.7.9 information, go to\\n5.7.9  https://support.google.com/mail/?p=WebLoginRequired d9443c01a7336-1f44c9e43c2sm4729765ad.308 - gsmtp')\n", "trace": "  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"Features\\environment.py\", line 108, in after_step\n    s.login(fromaddr, \"E$$pl@2022!#21\")\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 750, in login\n    raise last_exception\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 739, in login\n    (code, resp) = self.auth(\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 662, in auth\n    raise SMTPAuthenticationError(code, resp)\n"}, "attachments": [{"name": "screenshot", "source": "5e3ac45e-9512-4c1b-8574-5a6bbb6afa4f-attachment.png", "type": "image/png"}], "start": 1716527566276, "stop": 1716527571335}, {"name": "And Modify the \"consolidation ruleset check1\" and save", "status": "skipped", "start": 1716527571337, "stop": 1716527571337}, {"name": "And Click on Next Button", "status": "skipped", "start": 1716527571337, "stop": 1716527571337}, {"name": "And Click on Rule_Next Button", "status": "skipped", "start": 1716527571337, "stop": 1716527571337}, {"name": "And Click on Finish", "status": "skipped", "start": 1716527571337, "stop": 1716527571337}, {"name": "And Click on Confirm", "status": "skipped", "start": 1716527571337, "stop": 1716527571337}], "parameters": [{"name": "rulesetname", "value": "consolidation ruleset check1"}], "start": 1716527565270, "stop": 1716527571337, "uuid": "b1db8b65-60e4-4284-affa-141da4b3311a", "historyId": "4f7fd3711d0016e6218eaac267eaa841", "fullName": "Verify the TMS functionality: The user should be able to edit ruleset successfully.", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "Edit_ruleset"}, {"name": "feature", "value": "Verify the TMS functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}